torchvision/models/detection/retinanet.py

```diff
@@ -374,14 +426,23 @@ def __init__(self, backbone, num_classes,
         self._has_warned = False
 
     @torch.jit.unused
-    def eager_outputs(self, losses, detections):
+    def eager_outputs(
+        self,
+        losses: Dict[str, Tensor],
+        detections: List[Dict[str, Tensor]],
+    ):
```

Comment:
Bit confused here on the type of output.

Reply:
Yeah... this is an ugly workaround for JIT. See here for the output type:
https://github.com/pytorch/vision/blob/3e7653c8c4b5eb0c854251a0fb86be18de55bc68/torchvision/models/detection/ssd.py#L216-L217